pbs/gorson
# Updating Packages

this pr updates the packages to the latest using `go get -u all`... I also thought we should disable the cache for the tests as it's a bit counterintuitive for a library this small...

To test:
1) pull down this PR
2) run `go get all`
3) see tests pass `./scripts/test.sh`
